std/range/package.d

```diff
@@ -7172,25 +7172,25 @@ private struct OnlyResult(T, size_t arity)
 
     T front() @property
     {
-        assert(!empty);
+        assert(!empty, "Attempting to fetch the front of an empty only");
```

Comment:
We could switch to the convention: "<rangename>.front -- range is empty", e.g. "only.front -- range is empty"